package.json

```diff
@@ -72,7 +72,6 @@
 		"@release-it/conventional-changelog": "^10.0.0",
 		"@types/estree": "^1.0.6",
 		"@types/node": "^22.0.0",
-		"@types/package-json-validator": "^0.6.1",
```

Comment:
I guess this package should be marked as deprecated now?

Reply:
👍 agreed.